streamer/src/quic.rs

```diff
@@ -216,36 +267,124 @@ pub fn spawn_server(
                 )
                 .await;
 
+                if last_datapoint.elapsed().as_secs() >= 5 {
+                    datapoint_info!(
+                        "quic-connections",
+                        (
+                            "connections",
+                            total_connections.load(Ordering::Relaxed),
+                            i64
+                        ),
+                        ("streams", total_streams.load(Ordering::Relaxed), i64),
+                        (
+                            "new_connections",
+                            total_new_connections.swap(0, Ordering::Relaxed),
+                            i64
+                        ),
+                        (
+                            "new_streams",
+                            total_new_streams.swap(0, Ordering::Relaxed),
+                            i64
+                        ),
+                    );
+                    last_datapoint = Instant::now();
```

Comment:
also reset "new" counters?

Reply:
The report does a swap on the ones to reset.